src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,7 @@
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Method getAllProductsFromShoppingCart() successful fetched data from DB");
```

Comment:
The log message in the `getAllProductsFromShoppingCart` method should be more informative. Consider including the `userId` parameter to provide context, as suggested in the checklist: 'Let's make our log messages informative'.